hello_wifi/mix.exs

```diff
@@ -57,6 +57,11 @@ defmodule HelloWifi.Mixfile do
 
   # Specify target specific dependencies
   def deps("host"), do: []
+  def deps("rpi0") do
```

Comment:
`nerves_system_rpi0` is an "official" system now, so I think the `deps(target)` should work if we just bump the tag to `v0.12.0` rather than having a separate function head.